RELEASE.md

```diff
@@ -15,7 +15,7 @@ Release cadence of first pre-releases being cut is 6 weeks.
 | v2.8           | 2019-02-27                                 | Ganesh Vernekar (GitHub: @codesome)         |
 | v2.9           | 2019-04-10                                 | Brian Brazil (GitHub: @brian-brazil)        |
 | v2.10          | 2019-05-22                                 | Björn Rabenstein (GitHub: @beorn7)          |
-| v2.11          | 2019-07-03                                 | **searching for volunteer**                 |
+| v2.11          | 2019-07-03                                 | Julius Volz (GitHub: @juliusv)              |
```

Comment:
We should add the next few slots too

Reply:
I'd be ok with going again after, if nobody else wants to

Reply:
Krasi should do one, as well. 

Tom is probably too busy and will just delegate to his reportees.

We were thinking perhaps @codesome could do the next one.

So how about 2.11 @codesome , 2.12 @juliusv , 2,13 @krasi-georgiev , and then we check again later.

Reply:
@codesome did 2.8

Reply:
Updated. @codesome and @krasi-georgiev, please confirm.

Reply:
confirmed. I can't tell about my schedule around the time for 2.13, but will try to plan for it.

Reply:
I would probably skip 2.11 as I have been heavily out of touch since past couple of month due to academics. I can do any release after that.

Reply:
Yes, we know. @brancz Are you volunteering for 2.11?

Reply:
"Yes we know' was for @brancz comment. Concurrency, how does it work…

Reply:
Yeah I'm ok with doing 2.11.

Reply:
\o/ I'll update this PR.